src/index.ts

```diff
@@ -14,6 +14,32 @@ const CLICK_ID_COOKIE = `_${CLICK_ID_PARAM}`
 const CLICK_SOURCE_PARAM = 'clid_src'
 const ONE_MONTH = 2628000000
 
+async function hashUserData(event: MCEvent) {
+  const USER_DATA: Record<string, { hashed?: boolean }> = {
+    email_address: { hashed: true },
+    phone_number: { hashed: true },
+  }
+
+  const hashedData: Record<string, string> = {}
+  const encoder = new TextEncoder()
+
+  for (const [key, options] of Object.entries(USER_DATA)) {
```

Comment:
are we going to ever have `hashed: false` in USER_DATA? There's no need to have the options for now

Reply:
Yes we might, for example the `p_user_id` param. As agreed on the call, for now I keep it this way, so when we pick it up in the future it will be more organised.